docs/schema_markdown/Study.md

```diff
@@ -2,7 +2,7 @@
 # Class: Study
 
 
-Studies are experimental investigations of a particular phenomenon. It involves a detailed examination and analysis of a subject to learn more about the phenomenon being studied.
+A Study is an experimental investigation of a particular phenomenon. It involves a detailed examination and analysis of a subject to learn more about the phenomenon being studied.
```

Comment:
Study is an experimental investigation of a particular phenomenon. It involves detailed examination and analysis of a subject to learn more about the phenomenon being studied.